icu4j/main/tests/core/src/com/ibm/icu/dev/test/util/StringTokenizerTest.java

```diff
@@ -562,12 +562,12 @@ public void Test_GeneratePattern(){
             us._generatePattern(sb.append(1.0), true);
             us._generatePattern(sb.reverse(), true);
         } catch(Exception e){
-            errln("UnicodeSet._generatePattern is not suppose to return an exception.");
+            errln("UnicodeSet._generatePattern is not supposed to return an exception.");
```

Comment:
Couldn't help myself doing a replace-all on this misspelling...
Only one function has a real change. See around line 768.